### PR TITLE
feat: styleguide push

### PR DESCRIPTION
**Pull request by tu1ly**

### CODE REVIEW

**Comment by tu1ly** [comment]
[Apiary Constitutions](https://trello.com/c/THMFLuwi/299-apiary-constitutions)

File: lib/apiary/command/styleguide.rb [hunk 1]
```diff
@@ -196,11 +222,11 @@ def check_api_key
       abort 'Error: API key must be provided through environment variable APIARY_API_KEY. \Please go to https://login.apiary.io/tokens to obtain it.'
     end
 
-    def load
-      @add_path = api_description_source_path(@options.add)
-      @add = api_description_source(@add_path)
-      @functions = get_functions(@options.functions)
-      @rules = get_rules(@options.rules)
+    def load(add, functions, rules)
+      @add_path = api_description_source_path(@options.add) if add
```
Comment (JackuB) [inline lib/apiary/command/styleguide.rb]: Can't this be under one `if`?

File: lib/apiary/command/styleguide.rb [hunk 2]
```diff
@@ -68,7 +94,7 @@ def validate
       end
 
       begin
-        load
+        load(true, true, true)
```
Comment (JackuB) [inline lib/apiary/command/styleguide.rb]: This call looks really weird IMO - can't we use named arguments? Or pass an array of things we want?

File: lib/apiary/command/styleguide.rb (hunk 1 above)
Comment (tu1ly) [reply lib/apiary/command/styleguide.rb]: no - need false, true, true for push

File: lib/apiary/command/styleguide.rb (hunk 2 above)
Comment (tu1ly) [reply lib/apiary/command/styleguide.rb]: fixed

File: lib/apiary/command/styleguide.rb (hunk 1 above)
Comment (JackuB) [reply lib/apiary/command/styleguide.rb]: I mean there are 2 `if add`

Comment (tu1ly) [reply lib/apiary/command/styleguide.rb]: ah that one -doesn't matter (+ more lines)

Comment (JackuB) [reply lib/apiary/command/styleguide.rb]: giphy

It's a code smell

File: lib/apiary/version.rb [hunk 3]
```diff
@@ -1,3 +1,3 @@
 module Apiary
-  VERSION = '0.10.3'.freeze
+  VERSION = '0.11.3'.freeze
```
Comment (JackuB) [inline lib/apiary/version.rb]: btw isn't it `0.11.0'?

Comment (tu1ly) [reply lib/apiary/version.rb]: yup - fixed

**Comment by JackuB** [comment]
Please update the help page too

**Comment by tu1ly** [comment]
updated

**Pull request merged**